File: spring-webflux/src/main/java/victor/training/reactive/usecase/debugging/Apis.java
package victor.training.reactive.usecase.debugging;

import org.springframework.stereotype.Service;
import reactor.core.publisher.Mono;

@Service
public class Apis {

   public Mono<String> callGreen() {
      return Mono.just("1");
   }
   public Mono<String> callBlue() {
      return Mono.just("one");
   }
}

File: spring-webflux/src/main/java/victor/training/reactive/usecase/debugging/Apis.kt
package victor.training.reactive.usecase.debugging

import org.springframework.stereotype.Service
import reactor.core.publisher.Mono

@Service
class Apis {
    fun callGreen(): Mono<String> {
        return Mono.just("1")
    }

    fun callBlue(): Mono<String> {
        return Mono.just("one")
    }
}